easybuild/easyblocks/g/gromacs.py

```diff
@@ -202,8 +203,13 @@ def configure_step(self):
 
             (out, _) = run_cmd("plumed-patch -l", log_all=True, simple=False)
             if not re.search(engine, out):
-                raise EasyBuildError("There is no support in PLUMED version %s for GROMACS %s: %s",
-                                     get_software_version('PLUMED'), self.version, out)
+                msg = "There is no support in PLUMED version %s for GROMACS %s: %s" % (
+                    get_software_version('PLUMED'), self.version, out
+                )
```

Comment:
a bit less confusing w.r.t. formatting:

```suggestion
                plumed_ver = get_software_version('PLUMED')
                msg = "There is no support in PLUMED version %s for GROMACS %s: %s" % (plumed_ver, self.version, out)